resource/eastwood/config/third-party-libs.clj

```diff
@@ -286,3 +286,14 @@
   '([coll obj & {:keys [from to many write-concern]
                  :or {from :clojure to :clojure many false}}])
   :reason "somnium.congomongo/insert!  uses metadata to override the default value of :arglists for documentation purposes.  This configuration tells Eastwood what the actual :arglists is, i.e. would have been without that."})
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Configs to disable warnings in slingshot, version 0.12.2
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(disable-warning
+ {:linter :suspicious-expression
+  :for-macro 'clojure.core/and
+  :if-inside-macroexpansion-of #{'slingshot.slingshot/try+}
+  :within-depth 10
```

Comment:
https://github.com/scgilardi/slingshot/pull/55#issuecomment-215350429 indicates that `:within-depth 10` is not high enough.